index.html

```diff
@@ -1365,22 +1365,22 @@ <h3>Decoy Values</h3>
 [=Issuer=] use of decoy values in status lists has been explored as a mechanism
 to increase the privacy of [=subjects=]. While algorithms for employing decoy
 values are out of scope for this specification, implementers are advised that
-the use of decoy values do not provide privacy gains and can harm privacy in
+the use of decoy values does not improve privacy and can harm privacy in
 most cases.
       </p>
       <p>
-When status list entry indexes are allocated in a random fashion, which is the
-suggested mode of operation for this specification, adding decoys harms privacy
-because it reduces the group privacy size by the number of decoys added to the
-group. A random allocation of indexes inherently hides the true group size,
-ensuring that decoys are not necessary.
+When indexes of status list entries are allocated in a random fashion, which is
+the suggested mode of operation for this specification, adding decoys harms
+privacy because it reduces the group privacy size by the number of decoys added
+to the group. A random allocation of indexes inherently hides the true size of
+the group, ensuring that decoys are not necessary.
```

Comment:
```suggestion
privacy because it reduces the true group size by the number of decoys added
to the group. A random allocation of indexes inherently hides the true size of
the group without reducing it, ensuring that decoys are not necessary.
```